hashi/tasks/main.yml

```diff
@@ -7,3 +7,4 @@
 - name: Hashicorp repository
   ansible.builtin.apt_repository:
     repo: deb [arch=amd64] https://apt.releases.hashicorp.com {{ ansible_distribution_release }} main
+  update_cache: yes
```

Comment:
Does update_cache work for the repo? I thought this needed to be on the install step?